.funcignore

```diff
@@ -7,4 +7,4 @@ __blobstorage__
 __queuestorage__
```

Comment:
@dotnet-7 - Please add a .devcontainer for this project.  You can find examples in the official azd templates and modify it to suit the needs of this project.

Reply:
@jongio You really sure that `.devcontainer` is excluded by `.funcignore`?

Reply:
IC,  you already have a devcontainer in azd-template branch.  I was only looking in the preview branch.  I'm closing this.